src/extensions/default/AutoUpdate/main.js

```diff
@@ -275,11 +304,15 @@ define(function (require, exports, module) {
      * Sets up the Auto Update environment
      */
     function setupAutoUpdate() {
+        if (!checkIfAnotherSessionInProgress()) {
+            setAutoUpdateSessionInProgress(false);
```

Comment:
Session in progress is already false. Why are we setting this again?

Reply:
Addressed in 036e4a8